packages/media/src/createResource.js

```diff
@@ -121,6 +123,7 @@ function createResource({
   isPlaceholder = false,
   isOptimized = false,
   isMuted = false,
+  trim,
```

Comment:
Might be good to rename this to `trimData` to avoid confusion and express more clearly what this is. Just `trim` feels like a function without context.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/8972/commits/7527bbe5777d2946fb022410f05ebcc06f865e29